pontoon/base/static/js/translate.js

```diff
@@ -3962,7 +3973,7 @@ var Pontoon = (function (my) {
           queryParams = {};
 
       self.state = state;
-
+      self.updateTitle();
```

Comment:
You should add a blank line after this statement, to separate the block from the next one, which starts with a comment.

Actually, `pushState` is not the place to put `updateTitle` in - it's called everytime you open a new string for editing or load new entities, and we don't need to update the title on this occasion.

You should move it to the end of the `processEntities()` function.